spec.bs

```diff
@@ -271,7 +271,7 @@ To <dfn>check interest group permissions</dfn> given an [=origin=]
 1. Let |permissionsUrl| be the result of [=building an interest group permissions url=]
   with |ownerOrigin| and |frameOrigin|.
 1. Let |request| be the result of [=creating a request=] with |permissionsUrl|,
-  "`application/json`", and null.
+  "`application/json`", |frameOrigin|, and "`cors`".

```

Comment:
It _looks_ like this request is pretty much always cross-origin, right? If so, does our implementation make this request explicitly with CORS mode too? And if so, do we have any tests for the failure mode where the response to this request is *not* served with the proper CORS headers? I just want to make sure we're handling CORS side-effects here consistently with the implementation. Any WPTs would be great too.

(If I'm mistaken and this request is expected to be same-origin, maybe we can just make this `"same-origin"` and update the implementation accordingly; we did this with a request for ARA recently: https://github.com/WICG/attribution-reporting-api/pull/817).

Reply:
yes, same origin join/leaves are always [allowed](https://crsrc.org/c/content/browser/interest_group/interest_group_permissions_checker.cc;drc=adac219925ef5d9c9a954d189c2e4b8852a4bbed;l=96), and no need to fetch .well-known.
And our [implementation](https://crsrc.org/c/content/browser/interest_group/interest_group_permissions_checker.cc;drc=adac219925ef5d9c9a954d189c2e4b8852a4bbed;l=128) does explicitly set CORS mode. I did a quick exploration and didn't find a test where the response does not have a proper CORS header (still digging).